### ISSUE TITLE
feature: add Hugging Face agents support as new Argilla module

### ISSUE THREAD

**Issue by davidberenstein1957** [recognai/rubrix#3106]
**Is your feature request related to a problem? Please describe.**
Agents and LLMs are the cool business nowadays, and Hugging Face has a cool option to use them.

**Describe the solution you'd like**
Several custom tools that log info from other tools into Argilla.

**Describe alternatives you've considered**
N.A.

**Additional context**
https://huggingface.co/docs/transformers/custom_tools#adding-new-tools

**Comment by nataliaElv** [comment]
@tomaarsen suggested to tackle this after #3338

**Comment by tomaarsen** [comment]
This has a low priority as it does not make much sense before #3338 as mentioned by Natalia.
